evaluate_dyn_models.py

```diff
@@ -228,28 +228,23 @@ def onnx_faster_rcnn_layers():
         ((1, 507, 4), (1, 1, 507), 2000, 0.7, 0)
     ]
     return batch_norm, bias_add, nms
-    
+
 def generate_model_bn(dtype, input_shape, filter_shape, padding, strides, relu, leaky=False):
-    dtype = "float32"
+    dtype_init = "float32" if dtype == "float32" else "float16"
```

Comment:
nit: This check is a bit confusing -- if the user passes any dtype other than float32, float16 is used. Perhaps assert dtype is one of float32 or float16?

Reply:
This is merged, thanks @echuraev. Please consider a follow up PR.

Reply:
> nit: This check is a bit confusing -- if the user passes any dtype other than float32, float16 is used. Perhaps assert dtype is one of float32 or float16?

@csullivan We can pass three different types to the script: `float16`, `float16_acc32` and `float32`. So, `float16` should be used for `fp16` and mixed precision. This is why we have such condition here.